jenkins-pipelines/nonroot-offline-install/artifacts-rocky9.jenkinsfile

```diff
@@ -0,0 +1,15 @@
+#! groovy
+
+// trick from https://github.com/jenkinsci/workflow-cps-global-lib-plugin/pull/43
+def lib = library identifier: 'sct@snapshot', retriever: legacySCM(scm)
+
+artifactsPipeline(
+    test_config: 'test-cases/artifacts/rocky8.yaml',
```

Comment:
filename refers to rocky9 while here is rocky8

Reply:
fixing